scripts/variantstore/docs/aou/vds/removing_samples.md

```diff
@@ -0,0 +1,152 @@
+# Removing samples from the AoU VDS
+
+The following example walks through the procedure we followed for removing AI/AN samples from a preliminary version of
+the Delta VDS in November 2022. Because we did not have to re-create the filter with these samples removed, we did not
+need to re-run the `gvs_import.py` script. Instead we were able to load the previous version of the VDS, create a copy
+of the VDS with the samples in question filtered out, and then save this new VDS.
+
+## Configuring the Terra notebook cluster
+
+To work with a Delta-sized VDS (roughly 250K samples) we use our PMI ops accounts to create a cluster in a Terra
+notebook with the following configuration:
+
+![AoU Delta Cluster Config](./AoU Delta VDS Cluster Configuration.png)
+
+The startup script is at
+
+```
+gs://fc-secure-6e835ef6-2d1f-4ebf-9f21-313c0302f45b/patch_memory.sh
+```
+
+with contents:
+
+```shell
+#!/bin/sh
+printf "\nspark.driver.memory=85g\n" >> /etc/spark/conf/spark-defaults.conf
+```
+
+Note that this script hardcodes a memory value for the driver node. If using a driver node with a different amount of
+memory this value should be adjusted to avoid using too little or too much memory.
+
+## Installing the current Hail wheel
+
+Once the cluster is up and running, open a notebook terminal. Within the terminal copy down the latest Hail wheel:
+
+```
+$ gsutil -m cp gs://gvs-internal-scratch/hail-wheels/2022-10-18/0.2.102-964bee061eb0/hail-0.2.102-py3-none-any.whl .
+```
+
+Install the Hail wheel and its dependencies:
+
+```
+$ pip install --force-reinstall hail-0.2.102-py3-none-any.whl
+```
+
```

Comment:
Maybe something about ignoring the WARNINGs and red text that look alarming but don't matter?